client/package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "0.18.0",
     "firebase": "^5.6.0",
     "prop-types": "^15.6.2",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
     "react-redux": "^5.1.1",
+    "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.0",
     "redux": "^4.0.1",
-    "redux-thunk": "^2.3.0",
-    "axios": "0.18.0"
+    "redux-thunk": "^2.3.0"
```

Comment:
Nit. Looks like axios was moved up a few lines. I think the only real change is adding react-router-dom. Update the file so we just see the change to this package?

Reply:
npm automatically reorganizes the dependencies to alphabetical order when you do `npm install --save pkg`.

Reply:
I don't mind moving axios back, but should we keep it this way in case some one has to install another package in the future and it gets reshuffled again?

Reply:
I think we should just keep it as is (so no changes from the PR).

Reply:
Present revision?